blueflood-core/src/main/java/com/rackspacecloud/blueflood/tools/ops/FixTTL.java

```diff
@@ -0,0 +1,118 @@
+package com.rackspacecloud.blueflood.tools.ops;
+
+import org.apache.commons.cli.*;
+import com.netflix.astyanax.model.*;
+import com.rackspacecloud.blueflood.io.*;
+import com.rackspacecloud.blueflood.types.*;
+import java.util.*;
+
+public class FixTTL {
+    // Update all the TTL's for a set of metrics over a particular time range
+    public static void fixTTLs(ColumnFamily CF, String tenantID,
+                               List<String> metrics, Range range, Integer newTTL)
+            throws Exception {
+
+        List<Locator> locators = new ArrayList<Locator>();
+        for (String path: metrics) {
+            locators.add(Locator.createLocatorFromPathComponents(tenantID, path));
+        }
+
+        // Print out current TTL's, and update them
+        Map<Locator, ColumnList<Long>> data =
+                AstyanaxReader.getInstance().getColumnsFromDB(locators, CF, range);
```

Comment:
I am not sure if Astyanax paginates data, but what happens, when the supplied range is too long?